src/helper.ts

```diff
@@ -0,0 +1,13 @@
+import { Photo } from "./photo";
+
+export function getHtml(photos: Photo[]): string {
+    let result = "<div>"
+    for (let i = 0; i < photos.length; i++){
+        const url: string = photos[i].src;
+        const id: number= photos[i].id;
+        const photoStr: string = `<div> <img src = ${url} width = "200" height = "200"> <p>Picture id: ${id}</p></div>`;
+        result = result.concat(photoStr);
+    }
+    result = result.concat("</div>");
+    return result;
+}
```

Comment:
I like this function a lot! Can we give the function and the file a more descriptive name? It's okay if the file and the function have the same name!